packages/teleport/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "@gravitational/design": "1.0.0",
     "@gravitational/shared": "1.0.0",
+    "base64-arraybuffer": "^1.0.2",
```

Comment:
are we sure we need this? can you take something out of here: https://github.com/gravitational/webapps/blob/master/packages/shared/utils/base64url.ts 

i feel like we have multiple ways of decode/encoding base64

Reply:
I don't think that works because the strings are "standard base64" encoded, not "base64 url" encoded.

Reply:
yes but if you look at the code, it looks like we can just take out the two first lines of code?
```ts
export function base64urlToBuffer(baseurl64String: Base64urlString): ArrayBuffer {
  // Base64url to Base64
  const padding = '=='.slice(0, (4 - (baseurl64String.length % 4)) % 4);
  const base64String = baseurl64String.replace(/-/g, '+').replace(/_/g, '/') + padding;

  // Base64 to binary string
  const str = atob(base64String);

  // Binary string to buffer
  const buffer = new ArrayBuffer(str.length);
  const byteView = new Uint8Array(buffer);
  for (let i = 0; i < str.length; i++) {
    byteView[i] = str.charCodeAt(i);
  }
  return buffer;
}
```